### PR TITLE
Create translation deploy command

### PR DESCRIPTION
**Pull request by edulpn**

### CODE REVIEW

**Review by belesprengel — APPROVED** [review]
🚢🇮🇹

File: Sources/ios-cliKit/Command/Project/Translation/TranslationDeployCommand.swift [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    func execute() throws {
+        let languages = self.languages.compactMap { Languages(rawValue: $0) }
+
+        let dispatchGroup = DispatchGroup()
+
+        languages.forEach { language in
+
+            dispatchGroup.enter()
+
+            var request = URLRequest(url: URL(string: "https://api.poeditor.com/v2/projects/export")!)
```
Comment (txm5586) [inline Sources/ios-cliKit/Command/Project/Translation/TranslationDeployCommand.swift:46]: As discussed we could extract the make of the request to a method

File: Sources/ios-cliKit/Command/Project/Translation/TranslationDeployCommand.swift [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            URLSession.shared.dataTask(with: request) { [weak self] data, response, error in
+                guard let data = data,
+                      let exportedLanguage = try? JSONDecoder().decode(ExportedLanguage.self, from: data) else {
+                    self?.stdout <<< "Error exporting \(language.rawValue)\n"
+                    return
+                }
+
+                self?.stdout <<< "\(language.rawValue) exported\n"
+
+                try? Task.run(bash: "curl \(exportedLanguage.result.url) > strings.json.\(language.rawValue)")
```
Comment (txm5586) [inline Sources/ios-cliKit/Command/Project/Translation/TranslationDeployCommand.swift:67]: And make sure we get the exceptions and print the error

**Review by txm5586 — APPROVED** [review]
approved

**Pull request merged**